lib/elastomer/client/percolator.rb

```diff
@@ -0,0 +1,72 @@
+module Elastomer
+  class Client
+
+    class Percolator
+
+      # Create a new Percolator for managing a query.
+      #
+      # client - Elastomer::Client used for HTTP requests to the server
+      # index  - The index name
+      # id     - The _id for the query
+      def initialize(client, index, id)
+        @client = client
+        @index = index
+        @id = id
+      end
+
+      attr_reader :client, :index, :id
+
+      # Create a percolator query.
+      #
+      # Examples
+      #
+      #   percolator = $client.index("default-index").percolator "1"
+      #   percolator.create :query => { :match_all => { } }
+      #
+      # Returns the response body as a Hash
+      def create(body, params = {})
+        response = client.put("/#{@index}/.percolator/#{@id}", params.merge(:body => body, :action => 'percolator.create'))
```

Comment:
I'd prefer URI templates here, since that's the code path for all other path generation. See https://github.com/github/elastomer-client/blob/a19ffa5ab3207ecf162f1a37e70399bad1e31eb7/lib/elastomer/client/warmer.rb#L42.